cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1605,6 +1605,13 @@ endif()
 #---Check for protobuf-------------------------------------------------------------------
 
 if(tmva-sofie)
+  if(testing)
+    message(STATUS "Looking for BLAS as an optional testing dependency of TMVA-SOFIE")
+    find_package(BLAS)
```

Comment:
For SOFIE it is still fine to look for any BLAS backend?

Reply:
A bit related:
https://github.com/root-project/root/issues/16720
https://github.com/root-project/root/issues/16719
https://github.com/root-project/root/issues/16553